README.md

```diff
@@ -96,6 +96,30 @@ Endpoint: `Create a Self-Employment Period Summary`
 
 self-assessments-api `v2.0` has now been deprecated
 
+### 22 February 2023
+
+The following changes were deployed into sandbox.
+
+#### Individuals-charges-api
+
+Endpoint: `Create and Amend Pension Charges`
+
+* Created isAnnualAllowanceReduced data field that can be retrieved from pensionSavingsTaxCharges Object
```

Comment:
Could these all follow this pattern instead:

Fields added to the pensionSavingsTaxCharges object: isAnnualAllowanceReduced, moneyPurchasedAllowance, taperedAnnualAllowance